src/pages/Introduction/Introduction.tsx

```diff
@@ -1,18 +1,19 @@
 import React from "react";
 import Header from "../../components/Header/Header";
-import data from "../../../custom/data";
 import "./Introduction.scss";
 
 const Introduction: React.FC = () => {
-  const { introduction } = data;
-
   return (
     <div id="introduction" className="introduction">
       <Header />
       <div className="introduction-content">
-        <h1 className="introduction-text highlight-text">{introduction.title}</h1>
+        <h1 className="introduction-text light-text">
+          R<span className="highlight-text">AI</span>A
```

Comment:
Consider refactoring the component structure for improved readability and maintainability.

Suggested change:
```javascript
<h1 className="introduction-text light-text">
    R<span className="highlight-text">AI</span>A
</h1>
<p className="introduction-text-p light-text section-subtitle">
    Software
    <span className="highlight-text"> Developer •</span> Not a
    <span className="highlight-text"> Robot </span>
</p>
``` 

For reference, see [React Component Patterns](https://reactpatterns.com/).